src/frontend/src/components/ChangeRequestDropdown.tsx

```diff
@@ -33,9 +33,11 @@ const getFilteredChangeRequests = (changeRequests: ChangeRequest[], user: Authen
 interface ChangeRequestDropdownProps {
   control: Control<any, any>;
   name: string;
+  errors: FieldErrors<ChangeRequest>;
+  errorMessage?: FieldError;
```

Comment:
We can also delete the `errorMessage?` property here since we're not passing this property in anywhere in the codebase.